pkg/deploy/generator/rp.go

```diff
@@ -521,17 +520,16 @@ ExecStart=/usr/bin/docker run \
 ExecStop=/usr/bin/docker stop -t 3600 %N
 TimeoutStopSec="3600"
 Restart=always
-StandardError=null
 
 [Install]
 WantedBy=multi-user.target
 EOF
 
 cat >/etc/sysconfig/aro-monitor <<EOF
-MDM_ACCOUNT='$RPMDMMONITORINGACCOUNT'
-MDM_NAMESPACE='$RPMDMMETRICNAMESPACE'
-CLUSTER_MDM_ACCOUNT='$CLUSTERMDMMONITORINGACCOUNT'
-CLUSTER_MDM_NAMESPACE='$CLUSTERMDMMETRICNAMESPACE'
+MDM_ACCOUNT=AzureRedHatOpenShiftRP
```

Comment:
Will these not change and we leave them here. Im really on the fence when configurable are hardcoded in random places :/ they might not change but they should be in the single place. Maybe `env.StaticConfig() *StaticConfig` where we store all these and pass around?